Gemfile

```diff
@@ -74,7 +74,6 @@ gem "rails", "5.2.6.2"
 gem "rails-timeago", git: "https://github.com/librariesio/rails-timeago"
 gem "rb-readline"
 gem "rdoc"
-gem "RedCloth"
```

Comment:
Couldn't find a reference to RedCloth being used, even tho we do consume Textile in readmes.